server/bin/pbench-backup-tarballs.py

```diff
@@ -497,6 +498,11 @@ def main():
         return 1
 
     logger = get_pbench_logger(_NAME_, config)
+    try:
+        logging_level = config.get("pbench-backup-level", "logging_level")
```

Comment:
Why not `config.get(__NAME__, "logging_level")`?

Reply:
ok